ui/src/main/resources/FileManagerCode/DriveSheet.xml

```diff
@@ -1595,7 +1595,7 @@ require(['jquery', 'angular', 'JobRunner', 'angular-resource', 'angular-ui-boots
       scope: {
         'file': '='
       },
-      template: '&lt;img ng-src="{{icon}}" alt="{{type}}" title="{{type}}" class="icon" /&gt; '
+      template: '&lt;img ng-src="{{icon}}" alt="{{type}}" title="{{type}}" ng-class="(type === \'Image\') ? \'thumbnail-icon\' : \'icon\'"/&gt; '
```

Comment:
I'm not convinced that you need to use a different CSS class. It shouldn't matter whether the image is an icon or the file itself. We should display them both in the same way (with the same CSS).